app/controllers/admin/facilities_controller.rb

```diff
@@ -8,16 +8,42 @@ class Admin::FacilitiesController < AdminController
   before_action :initialize_upload, :validate_file_type, :validate_file_size, :parse_file,
     :validate_facility_rows, if: :file_exists?, only: [:upload]
 
+  skip_after_action :verify_authorized, if: -> { Flipper.enabled?(:new_permissions_system_aug_2020, current_admin) }
+  skip_after_action :verify_policy_scoped, if: -> { Flipper.enabled?(:new_permissions_system_aug_2020, current_admin) }
+  after_action :verify_access_authorized, if: -> { Flipper.enabled?(:new_permissions_system_aug_2020, current_admin) }
+
   def index
-    authorize([:manage, :facility, Facility])
+    if Flipper.enabled?(:new_permissions_system_aug_2020, current_admin)
+      current_admin.authorize(:manage, :facility)
+    else
+      authorize([:manage, :facility, Facility])
+    end
 
     if searching?
-      facilities = policy_scope([:manage, :facility, Facility]).search_by_name(search_query)
+      facilities = if Flipper.enabled?(:new_permissions_system_aug_2020, current_admin)
+        current_admin.accessible_facilities(:manage).search_by_name(search_query)
+      else
+        policy_scope([:manage, :facility, Facility]).search_by_name(search_query)
+      end
+
       facility_groups = FacilityGroup.where(facilities: facilities)
 
       @organizations = Organization.where(facility_groups: facility_groups)
       @facility_groups = facility_groups.group_by(&:organization)
       @facilities = facilities.group_by(&:facility_group)
+    elsif Flipper.enabled?(:new_permissions_system_aug_2020, current_admin)
```

Comment:
standardrb complains about nested conditionals, turned it into an `elsif` even though its a little harder to read